inscriptions/ISic3185.xml

```diff
@@ -177,25 +182,30 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI329579">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                     Εὐστάχις ἡ κὲ | Βῶ|κυς<lb n="2"/> ἐκοιμήθη<lb n="3"/> εἰδοῖς Μαείες <surplus>
-                    ²⁶Μαΐαις</surplus>²⁶<lb n="4"/>σώφρων καὶ ἀκα-<lb n="5"/>τάγνωστος· ἔζησεν<lb n="6"/> ἔτη λεʹ<lb n="7"/>καὶ ἀνεπαύσατο.
-                    <lb n="8"/>
+                    <lb n="1"/>Εὐστάχις <hi rend="ligature">ἡ</hi> κ<choice><corr>αὶ</corr><sic>ὲ</sic></choice> Βῶκυς
+                    <lb n="2"/>ἐκοιμήθη
+                    <lb n="3"/>εἰδοῖς Μαείες
+                    <lb n="4"/>σώφρων καὶ ἀκα
+                    <lb n="5" break="no"/>τάγνωστος ἔζησεν
+                    <lb n="6"/>ἔτη <num value="35">λε</num>
+                    <lb n="7"/>καὶ ἀνεπαύσατο
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after Orsi 1923</note>
+                    </app>
+                </listApp>
             </div>
             <div type="translation">
                 <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
             </div>
             <div type="commentary">
                 <p><!--commented out pending revision-->
-                    <!--Eustachis he ke (i.e. kai) Bokys (bwkus) ekoimethe eidois Maeies (i.e. Maiais) sophron kai akatagnostos; ezesen ete le' kai anepausato.--></p>
+                    <!--Eustachis he ke (i.e. kai) Bokys (bwkus; i.e. Bokchos) ekoimethe eidois Maeies (i.e. Maiais) sophron kai akatagnostos; ezesen ete le' kai anepausato. The name Bokys is written smaller between l.2 and 3. It does not seem to be related to the rest of the text.--></p>
```

Comment:
NB this is the interpretation of Ferrua, probably more convincing than Orsi's, and that it should all eb read as an addition marked as hi rend="small" on line 1 seems best, with a note in the apparatus that it adjoins the ends of tlines 1-3